src/compounds/product-table/src/components/cell-image.tsx

```diff
@@ -4,9 +4,17 @@ import { jsx } from 'theme-ui'
 
 import CellBase from './cell-base'
 
-export const ProductTableCellImage: React.FC = ({ children }) => (
+export interface ProductTableCellImageProps extends React.HTMLAttributes<any> {
```

Comment:
These aren't following convention - `CellImageProps` to be more similar to the other cell types (all the other ones in this PR are also wrong)